### PR TITLE
SEO-86912-webforms-redirect-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @Yvone-Atieno,

|Title|Description|
|--|--|
|Category|permanent redirect link |
|Hotfix PR|https://github.com/syncfusion-content/asp.netwebforms-docs/pull/542|
|Task||
|Content Ticket||
|UX Ticket||

|Changes made|Reason for changes|
|--|--|
|Changed redirect links|To avoid redirect link issues in the page and add direct link |
|Fixed meta elements CI issue| To provide proper meta alignment|

Regards,
Edith.

### CODE REVIEW

File: aspnet/SpreadSheet/Editing.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Editing with Spreadsheet widget for Syncfusion Essential JS
-description: Learn about editing support in Syncfusion ASP.NET Web Forms Spreadsheet control, its elements, and more details. 
+title: Editing in the ASP.NET Web Forms Spreadsheet Control | Syncfusion
+description: Learn here about editing support in Syncfusion Essential ASP.NET Webforms Spreadsheet control, its elements and more details.
```
Comment (Yvone-Atieno) [inline aspnet/SpreadSheet/Editing.md:4]: Update as Learn here about editing support in Syncfusion Essential ASP.NET Web Forms Spreadsheet control, its elements, and more details.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/SpreadSheet/Editing.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Editing with Spreadsheet widget for Syncfusion Essential JS
-description: Learn about editing support in Syncfusion ASP.NET Web Forms Spreadsheet control, its elements, and more details. 
+title: Editing in the ASP.NET Web Forms Spreadsheet Control | Syncfusion
```
[the remark this answers is not in the document]
Comment (Edithabich) [reply aspnet/SpreadSheet/Editing.md]: resolved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**